pyanaconda/display.py

```diff
@@ -50,6 +50,17 @@
 stdout_log = get_stdout_logger()
 
 
+X_FAILURE_MESSAGE = \
+    "X startup failed, falling back to text mode. There are multiple ways to help this issue:\n" \
```

Comment:
"There are multiple ways to help this issue" sounds to me like the bullet points above help _cause_ the issue.

Instead I recommending changing the formulation to something like this:

"There are multiple ways that might help avoiding this issue"